packages/orama/src/methods/search.ts

```diff
@@ -107,7 +107,10 @@ export async function search(orama: Orama, params: SearchParams, language?: stri
     const propertiesToSearchWithTypes = await orama.index.getSearchablePropertiesWithTypes(index)
 
     propertiesToSearch = await orama.index.getSearchableProperties(index)
-    propertiesToSearch = propertiesToSearch.filter((prop: string) => propertiesToSearchWithTypes[prop] === 'string')
+    propertiesToSearch = propertiesToSearch.filter(
+      (prop: string) =>
+        propertiesToSearchWithTypes[prop] === 'string' || propertiesToSearchWithTypes[prop] === 'string[]',
```

Comment:
The naive me would have refactored this into `['string', 'string[]'].includes(propertiesToSearchWithTypes[prop])`, now I am not sure if I like it or not ahah

Reply:
Even a RegExp :)

Reply:
or startsWith('string') ?

Reply:
Done using `.startsWith('string')`.